[2024-05-04T02:50:43.056+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_analytics_dag.transform_data manual__2024-05-04T02:50:30.953543+00:00 [queued]>
[2024-05-04T02:50:43.064+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_analytics_dag.transform_data manual__2024-05-04T02:50:30.953543+00:00 [queued]>
[2024-05-04T02:50:43.065+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-04T02:50:43.065+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2024-05-04T02:50:43.066+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-04T02:50:43.076+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): transform_data> on 2024-05-04 02:50:30.953543+00:00
[2024-05-04T02:50:43.081+0000] {standard_task_runner.py:55} INFO - Started process 325 to run task
[2024-05-04T02:50:43.087+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'covid_analytics_dag', 'transform_data', 'manual__2024-05-04T02:50:30.953543+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/covid_analytics.py', '--cfg-path', '/tmp/tmpgdm0m3n9']
[2024-05-04T02:50:43.088+0000] {standard_task_runner.py:83} INFO - Job 91: Subtask transform_data
[2024-05-04T02:50:43.164+0000] {task_command.py:376} INFO - Running <TaskInstance: covid_analytics_dag.transform_data manual__2024-05-04T02:50:30.953543+00:00 [running]> on host da20c7caeb7d
[2024-05-04T02:50:43.232+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Diego Munera
AIRFLOW_CTX_DAG_ID=covid_analytics_dag
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-04T02:50:30.953543+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-04T02:50:30.953543+00:00
[2024-05-04T02:50:43.247+0000] {credentials.py:1180} INFO - Found credentials in environment variables.
[2024-05-04T02:50:52.296+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-05-04T02:50:52.334+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=covid_analytics_dag, task_id=transform_data, execution_date=20240504T025030, start_date=20240504T025043, end_date=20240504T025052
[2024-05-04T02:50:52.409+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-04T02:50:52.440+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
